plugin/assets/src/block-editor/blocks/card/components/inspector-controls-style-panel.js

```diff
@@ -42,6 +42,17 @@ const CONTENT_LAYOUTS = [
 	},
 ];
 
+const CARD_ELEVATION_STYLES = [
+	{
+		label: __( 'Outlined', 'material-design' ),
+		value: 'outlined',
+	},
+	{
+		label: __( 'Filled', 'material-design' ),
+		value: 'filled',
```

Comment:
Instead of filled, Should we say elevated as per material specs? https://material.io/components/cards